src/coprocessor/dag/rpn_expr/mod.rs

```diff
@@ -10,29 +10,99 @@ mod impl_op;
 pub use self::function::RpnFunction;
 pub use self::types::{RpnExpression, RpnExpressionBuilder};
 
-use tipb::expression::ScalarFuncSig;
-
-use crate::coprocessor::Result;
+use cop_datatype::{FieldTypeAccessor, FieldTypeFlag};
+use tipb::expression::{Expr, ScalarFuncSig};
 
 use self::impl_compare::*;
 use self::impl_op::*;
+use crate::coprocessor::codec::data_type::*;
+use crate::coprocessor::Result;
+
+fn map_compare_int_sig<F: CmpOp>(
+    value: ScalarFuncSig,
+    children: &[Expr],
+) -> Result<Box<dyn RpnFunction>> {
+    // FIXME: The signature for different signed / unsigned int should be inferred at TiDB side.
+    if children.len() != 2 {
+        return Err(box_err!(
+            "ScalarFunction {:?} (params = {}) is not supported in batch mode",
```

Comment:
Is RPN function just be used in batch mode? The error message should be refined if RPN function will be used in more widely scenaoirs.

Reply:
It is only used in batch.